pkg/util/storage/manager.go

```diff
@@ -43,7 +46,22 @@ func (m *manager) BlobService(ctx context.Context, resourceGroup, account string
 		SharedAccessStartTime:  &date.Time{Time: t},
 		SharedAccessExpiryTime: &date.Time{Time: t.Add(24 * time.Hour)},
 	})
+
 	if err != nil {
+		if strings.Contains(err.Error(), "TooManyRequests") {
+			err = &api.CloudError{
+				StatusCode: http.StatusTooManyRequests,
+				CloudErrorBody: &api.CloudErrorBody{
+					Code:    api.CloudErrorCodeThrottlingLimitExceeded,
+					Message: "ThrottlingLimitExceeded",
+					Details: []api.CloudErrorBody{
+						{
+							Message: string("Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details."),
```

Comment:
LGTM, a small nit,

```suggestion
							Message: string("Requests are throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot#exceeding-azure-storage-limits for more details."),
```

Reply:
We generated this error message with reference to the original message that we go from Azure aside : "The request is being throttled as the limit has been reached for operation type - Read_ObservationWindow_00:05:00. For more information, see - https://aka.ms/srpthrottlinglimits"